### ISSUE TITLE
December 2022 Contributions - {ChrisBck#1751}

### ISSUE THREAD

**Issue by chrisbck** [Phala-Network/growth-program#92]
Telegram : ChrisBck
Discord : ChrisBck#1751
PHA Address : 43ZKJ2DEBagpMyn4EQvTu7niH1xRYGvfCJyGZYZyW6STFCbd
My Mandatory (Basic) Contributions of December:
Candidate / PhalaWorld Night’s Watch

Have I participated in Discord Channel Moderation ? yes
Have I participated in Telegram Moderation ? no

---

## Twitter

Twitter: https://twitter.com/BckChris
![twitterHomeDec23](https://user-images.githubusercontent.com/32775411/210691182-ad788c2e-d188-412d-84a6-f1fec5569b3a.png)
![twittertweetsDec23](https://user-images.githubusercontent.com/32775411/210691195-b64eaff9-fe56-47c5-aed4-5712636984cb.png)

<br>

## CM

**PhalaWorld**

I have been mainly active and focussed in the PhalaWorld discord. December was a very busy month in the community with the food incubation and dress-up events. This required a lot of fault finding and guidance to help make the process as smooth as possible, therefore a lot of extra hours were put in daily to make sure we had as much support coverage of each day as possible.

<br>

## HD

I helped resolve many issues and provided a lot of guidance during dressup and feeding events. I also found the root of an issue which caused feeding problems due to wallets not allowing phalaworld.com and phala.world endpoints. 

![PW feeding issue FAQ](https://user-images.githubusercontent.com/32775411/210695229-e73747f7-33ac-407e-b0af-cea066c2d077.png)

<br>

## CC

I did not contribute much as in this field as was primarily focussed on community duties. I did make a short video with some basic vfx for the dress-up:
https://twitter.com/BckChris/status/1605093538273714177

and a Christmas Greeting image:
https://twitter.com/BckChris/status/1606685471169744901

<br>

---

In summary, my contributions are quite thin outwith of CM. As I am new to this program I did not have these tasks assigned for most of the month. Therefore my scoring for this month is quite low. Apologies if I have not got this correct but I am gauging my scores as follows:

CM: 50
CC: 5
HD: 10
Total: 65
